tools/manifest-generator/manifest-generator.go

```diff
@@ -234,7 +257,7 @@ func generateFromFile(templFile string) {
 func getCdiLogo(path string) string {
 	file, err := os.Open(path)
 	if err != nil {
-		klog.Fatalf("Error retrieving cdi logo file: %v\n", err)
+		klog.Fatalf("Error retrieving cdi logo file: %v, %s\n", err, path)
```

Comment:
I would flip the %s and %v, to be consistent with the one above.

Reply:
Done